test_plans/pidgin/test_plan.md

```diff
@@ -0,0 +1,41 @@
+Pidgin is a light-weight API on which works on top of Peregrine. It takes a file's GUID as input which queries Peregrine for information about the file and returns an abstract of the file to the user.
+
+Integration Test Coverage for Pidgin :
+Pidgin has a single endpoint  - /<GUID of a file>
+
+Endpoints :	
+
+1. /<GUID of file> - this endpoint queries Peregrine and returns core metadata about the file. By default, it returns core metadata as JSON document.
+                three formats are supported by this endpoint - JSON, bibliography, and JSON-LD. The user should specify the format of the output with the help of "Accept" header. 
+
+    a. Accept: application/json # for JSON format	
+       the output of the JSON format
+       `{"file_name": "my-file.txt", "data_format": "TXT", "file_size": 10, "object_id": "123"}`
```

Comment:
for all the paragraphs, you need to add a blank line in-between each new line, or markdown displays it on a single line :/
https://github.com/uc-cdis/gen3-qa/blob/b381734346ee8ca3fa8d2de6f491406ec6110da3/test_plans/pidgin/test_plan.md